src/decls/types.js

```diff
@@ -93,3 +93,22 @@ declare type TChannelSpan = {
     parentid: Array<number>,
     flags: number
 };
+
+declare type ReporterConfig = {
+    flushIntervalMs: ?number,
```

Comment:
is milliseconds consistent with the other languages? I know in Go the value is actually typed. In Python it's probably a float as seconds, which seems the most sensible default scale